xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -348,3 +376,36 @@ void CAndroidUtils::OnSettingChanged(const std::shared_ptr<const CSetting>& sett
   if (settingId == CAndroidUtils::SETTING_LIMITGUI)
     CDisplaySettings::GetInstance().ClearCalibrations();
 }
+
+std::vector<int> CAndroidUtils::GetDisplaySupportedHdrTypes()
+{
+  CJNIWindow window = CXBMCApp::getWindow();
+  std::vector<int> hdrTypes;
+
+  if (window)
+  {
+    CJNIView view = window.getDecorView();
+    if (view)
+    {
+      CJNIDisplay display = view.getDisplay();
+      if (display)
+      {
+        CJNIDisplayHdrCapabilities caps = display.getHdrCapabilities();
+        if (caps)
+          hdrTypes = caps.getSupportedHdrTypes();
+      }
+    }
+  }
+
+  return hdrTypes;
+}
+
+void CAndroidUtils::LogDisplaySupportedHdrTypes()
+{
+  std::vector<int> hdrTypes = CAndroidUtils::GetDisplaySupportedHdrTypes();
+
+  for (int type : hdrTypes)
+  {
+    CLog::Log(LOGDEBUG, "CAndroidUtils: Display Supported HDR type: {}", HdrTypeString(type));
```

Comment:
i suggest to collect the strings to print in a single debug line instead to print a new log line for each type, the log is already heavy

Reply:
Thanks, done.
Now only one line at init is printed: `DEBUG <general>: CAndroidUtils: Display Supported HDR types: Dolby Vision HDR10 HLG `